app/lib/package/export_api_to_cdn.dart

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Should we call the file `export_api_to_bucket` instead? As the cdn is somewhat orthogonal.